### DISCUSSION TITLE
【福島系列三】鐵血廢核　福島要靠再生能源翻身

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#704] [General]
https://futurecity.cw.com.tw/article/527
如何有效的發展能源，不再看到烙烙長的公式跟專家分析，而是一個軟體幫妳我解決問題，說實在話的很多人對於能源危機沒有太多的想法，畢竟在它們心中，這就是給專家去想就好的議題，但最近氣候的變化，相信大家都有目共睹，這樣的情況，不能再只有，全球的少數人去想，應該要變成一種全體的意識，而這個方法就是給我們這種老百姓的最佳解

**Comment by yjpxi** [comment]
雖然這類軟體旨在讓更多人參與能源問題的解決，但並非所有人都願意或能夠使用這些技術，特別是技術知識相對薄弱的群體，因此，這類工具可能未必能夠實現其預期的廣泛影響，導致部分人群仍然對能源問題漠不關心。
